src/clients/java/ci.zig

```diff
@@ -33,11 +33,73 @@ pub fn tests(shell: *Shell, gpa: std.mem.Allocator) !void {
     }
 }
 
-pub fn verify_release(shell: *Shell, gpa: std.mem.Allocator, tmp_dir: std.fs.Dir) !void {
-    var tmp_beetle = try TmpTigerBeetle.init(gpa, .{});
+pub fn validate_release(shell: *Shell, gpa: std.mem.Allocator, options: struct {
+    version: []const u8,
+    tigerbeetle: []const u8,
+}) !void {
+    var tmp_beetle = try TmpTigerBeetle.init(gpa, .{
+        .prebuilt = options.tigerbeetle,
+    });
     defer tmp_beetle.deinit(gpa);
 
-    // TODO
-    _ = shell;
-    _ = tmp_dir;
+    try shell.env.put("TB_ADDRESS", tmp_beetle.port_str.slice());
+
+    try std.fs.cwd().writeFile(
+        "pom.xml",
+        \\<project>
+        \\  <modelVersion>4.0.0</modelVersion>
+        \\  <groupId>com.tigerbeetle</groupId>
+        \\  <artifactId>samples</artifactId>
+        \\  <version>1.0-SNAPSHOT</version>
+        \\
+        \\  <properties>
+        \\    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        \\    <maven.compiler.source>11</maven.compiler.source>
+        \\    <maven.compiler.target>11</maven.compiler.target>
+        \\  </properties>
+        \\
+        \\  <build>
+        \\    <plugins>
+        \\      <plugin>
+        \\        <groupId>org.apache.maven.plugins</groupId>
+        \\        <artifactId>maven-compiler-plugin</artifactId>
+        \\        <version>3.8.1</version>
+        \\        <configuration>
+        \\          <compilerArgs>
+        \\            <arg>-Xlint:all,-options,-path</arg>
+        \\          </compilerArgs>
+        \\        </configuration>
+        \\      </plugin>
+        \\
+        \\      <plugin>
+        \\        <groupId>org.codehaus.mojo</groupId>
+        \\        <artifactId>exec-maven-plugin</artifactId>
+        \\        <version>1.6.0</version>
+        \\        <configuration>
+        \\          <mainClass>com.tigerbeetle.samples.Main</mainClass>
+        \\        </configuration>
+        \\      </plugin>
+        \\    </plugins>
+        \\  </build>
+        \\
+        \\  <dependencies>
+        \\    <dependency>
+        \\      <groupId>com.tigerbeetle</groupId>
+        \\      <artifactId>tigerbeetle-java</artifactId>
+        \\      <version>0.14.158</version>
```

Comment:
Shouldn't we use the targetted version here?

```suggestion
        \\      <version>{version}</version>
```

Reply:
Most excellent catch, thanks!

Fun fact, when writing this code I've actually thought that `options: struct` pattern has a drawback that it doesn't warn about unused field, as I made a similar mistake for some other language!